{{cookiecutter.project_slug}}/setup_env.sh

```diff
@@ -1,61 +1,109 @@
 #!/bin/bash
+#
+# Create conda environment with pinned or unpinned run and/or dev requirements
+#
+# - 2022-08 (D. Regenass) Write original script
+# - 2022-09 (S. Ruedisuehli) Refactor; add some options
+#
 
-### OPTION DEFAULTS ###
-ENV_NAME={{cookiecutter.project_slug}}
+# Default options
+ENV_NAME="{{ cookiecutter.project_slug.replace("_", "-") }}"
 PYVERSION=3.10
+PINNED=true
 DEV=false
-PINNED=false
-#GET OPTIONS FROM COMMAND LINE ARGS
-while getopts n:v:dph flag
-do
+EXPORT=false
+INSTALL=false
+FORCE=false
+CONDA=conda
+HELP=false
+
+help="Usage: $(basename "${0}") [-n NAME] [-P VER] [-u] [-e] [-d] [-i] [-f] [-m] [-h]
+
+Options:
+ -n NAME    Env name (-d adds -dev) [default: ${ENV_NAME}]
+ -P VER     Python version [default: ${PYVERSION}]
+ -u         Use unpinned requirements (minimal version restrictions)
```

Comment:
with the additional options we are introducing here, we are introducing also complexity, so I am not sure it is worth for the user. 
In particular not sure about the force (I would simply assume the user always starts with a clean env and later on we can use pip for installing/updating its package)
Also the export (alternative is to use python tools, i.e. conda for that) and the -i ( I would assume you always wants to install the package. For individual/partial operations user rather conda standalone).

Reply:
I have a slight tendency (but no strong opinion really) to go with @cosunae 's point of view. Both the` -f` and the `-i` options were at some point implemented in the first draft but after a discussion with both @cosunae and the DA team, we decided to keep the script as simple as possible and to separate concerns, that are (a) set up miniconda (b) install the requirements (c) install the package. Therefore we abandoned these options. As for the conda env export, I think that would be reasonable to have in this script as it is heavily related to the creation of the environment.

Reply:
Removed `-f` and `-i`, but kept `-e`.